integration-tests/tracecontext/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3@sha256:589ed6659c0e4aac182f309131cd35e85452d21072570b1f6abc45b7687093a3 AS build
+FROM python:3.13.2@sha256:589ed6659c0e4aac182f309131cd35e85452d21072570b1f6abc45b7687093a3 AS build
```

Comment:
the changes in this file are unrelated to the frequency, but I think it's nice to see the version numbers (and it won't make it any more frequent since the hashes will get updated on every release anyways)